cmd/stats-exporter/main.go

```diff
@@ -0,0 +1,114 @@
+package main
+
+import (
+	"database/sql"
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/letsencrypt/boulder/cmd"
+)
+
+type sqlRows interface {
+	Next() bool
+	Scan(dest ...interface{}) error
+}
+
+type dbQueryable interface {
+	Query(string, ...interface{}) (*sql.Rows, error)
+}
+
+var sqlOpen = func(driver, dsn string) (dbQueryable, error) {
+	return sql.Open(driver, dsn)
+}
+
+var execRun = func(c *exec.Cmd) error {
+	return c.Run()
+}
+
+func queryDB(dbConnect, beginTimeStamp, endTimeStamp string) (*sql.Rows, error) {
+	dbDSN, err := ioutil.ReadFile(dbConnect)
+	if err != nil {
+		return nil, fmt.Errorf("Could not open database connection file: %s", err)
```

Comment:
Can you include the `dbConnect` filename in the error message? I think the most likely cause of failure here is if the file doesn't exist or isn't readable and including the file name will make troubleshooting faster/easier.
```suggestion
		return nil, fmt.Errorf("Could not open database connection file %q: %s", dbConnect, err)
```